results/custom_example_location.tsv

```diff
@@ -1,101 +1,101 @@
 "snp"	"refsnp_source"	"chrom"	"position"	"snp_end"	"minor_allele"	"group"
```

Comment:
I'm not a big fan of these quotes (unless the field actually contains a tab).

May not be the right PR to change this.